integration-tests/package.json

```diff
@@ -13,7 +13,7 @@
     "author": "",
     "license": "Apache-2.0",
     "dependencies": {
-        "@frequency-chain/api-augment": "v0.9.29",
+        "@frequency-chain/api-augment": "^v0.9.29",
```

Comment:
This is mostly done as .9.30 wont be available until release, so pinning on last stable release

Reply:
I need to make it so that for CI this builds and uses the "local" version so they stay in sync for ci. On my need a break list ;)